plugins/font-awesome.user.js

```diff
@@ -0,0 +1,30 @@
+// ==UserScript==
+// @id             iitc-plugin-font-awesome@Zaso
+// @name           IITC plugin: Font Awesome
```

Comment:
If we include web font - we should include it in core, to make it available out-of-the-box.